bears/natural_language/WriteGoodLintBear.py

```diff
@@ -0,0 +1,68 @@
+from coalib.bearlib.abstractions.Linter import linter
+from coalib.bears.requirements.NpmRequirement import NpmRequirement
+
+
+@linter(executable='write-good',
+        output_format='regex',
+        output_regex=r'(?P<message>.*)\s*on\s*line\s*(?P<line>\d+)\s*at\scolumn'
+                      '\s*(?P<column>\d+)'
+        )
+class WriteGoodLintBear:
+    """
+    Lints the text files using ``write-good`` for improving proses.
+
+    See <https://github.com/btford/write-good> for more information.
+    """
+    LANGUAGES = {"Natural Language"}
+    REQUIREMENTS = {NpmRequirement('write-good', '0.9.1')}
+    AUTHORS = {'The coala developers'}
+    AUTHORS_EMAILS = {'coala-devel@googlegroups.com'}
+    LICENSE = 'AGPL-3.0'
+    CAN_DETECT = {'Formatting', 'Grammar'}
+
+    @staticmethod
+    def create_arguments(filename, file, config_file,
+                         use_passive_voice: bool=False,
+                         check_so_beginning: bool=False,
+                         check_adverbs: bool=False,
+                         check_repeated_words: bool=False,
+                         check_is_beginning: bool=False,
+                         check_ambiguos_words: bool=False,
+                         check_extra_words: bool=False,
+                         check_cliche_phrases: bool=False):
+        """
+        Using ``False`` will disable the check.
+
+        :param use_passive_voice:       Checks for passive voice.
+        :param check_so_beginning:      Checks for ``So`` at the beginning of
+                                        the sentence.
+        :param check_adverbs:           Checks for adverbs that can weaken the
+                                        meaning, such as: ``really``, ``very``,
+                                        ``extremely``, etc.
+        :param check_repeated_words:    Checks for lexical illusions – cases
+                                        where a word is repeated.
+        :param check_is_beginning:      Checks for ``There is`` or ``There are``
```

Comment:
better variable name? AFAIR the previous name you had for this setting was better what happened? maybe `check_beginning` or `check_there_is`